plugins/woocommerce/src/Blocks/BlockTypes/ProductSpecifications.php

```diff
@@ -102,25 +102,46 @@ public function render( $attributes, $content, $block ) {
 		ob_start();
 
 		$wrapper_attributes = get_block_wrapper_attributes(
-			array( 'class' => 'wc-block-product-specifications' )
+			array( 'class' => 'wp-block-table' )
 		);
 		?>
-		<table <?php echo $wrapper_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?> aria-label="<?php esc_attr_e( 'Product Specifications', 'woocommerce' ); ?>">
-			<tbody>
-				<?php foreach ( $product_data as $product_attribute_key => $product_attribute ) : ?>
-					<tr class="wc-block-product-specifications-item wc-block-product-specifications-item__<?php echo esc_attr( $product_attribute_key ); ?>" scope="row">
-						<th class="wc-block-product-specifications-item__label">
-							<?php echo wp_kses_post( $product_attribute['label'] ); ?>
-						</th>
-						<td class="wc-block-product-specifications-item__value">
-							<?php echo wp_kses_post( $product_attribute['value'] ); ?>
-						</td>
+		<figure <?php echo $wrapper_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
+			<table>
+				<thead class="screen-reader-text">
+					<tr>
+						<th><?php esc_html_e( 'Attributes', 'woocommerce' ); ?></th>
+						<th><?php esc_html_e( 'Value', 'woocommerce' ); ?></th>
 					</tr>
-				<?php endforeach; ?>
-			</tbody>
-		</table>
+				</thead>
+				<tbody>
+					<?php foreach ( $product_data as $product_attribute_key => $product_attribute ) : ?>
+						<tr class="wc-block-product-specifications-item wc-block-product-specifications-item__<?php echo esc_attr( $product_attribute_key ); ?>" scope="row">
+							<td class="wc-block-product-specifications-item__label">
+								<?php echo wp_kses_post( $product_attribute['label'] ); ?>
+							</td>
+							<td class="wc-block-product-specifications-item__value">
+								<?php echo wp_kses_post( $product_attribute['value'] ); ?>
+							</td>
+						</tr>
+					<?php endforeach; ?>
+				</tbody>
+			</table>
+		</figure>
 		<?php
 
 		return ob_get_clean();
 	}
+
+	/**
+	 * Get the frontend style handle for this block type.
+	 *
+	 * @return string[]|null
```

Comment:
I think that
```suggestion
	 * @return string[]
```

Should we update this line for the `AbstractBlock` class too?

https://github.com/woocommerce/woocommerce/blob/e28f1ded48ace7ddaaddf5c564e6cc684f957a02/plugins/woocommerce/src/Blocks/BlockTypes/AbstractBlock.php#L359-L370

Reply:
I think we shouldn't update the abstract class, it indicates that the child block can return null to disable block style.

Reply:
Good point! It makes sense 👍